### DISCUSSION TITLE
RHF weird focus behavior with react select

### DISCUSSION THREAD

**Discussion by spector163** [react-hook-form/react-hook-form#10126] [Q&A]
Hello, 
     I have a question regarding focus management on errors in RHF. React-Select is not focused on errors if any uncontrolled input in RHF has Error. Even if React select comes first in the render function. Can anybody help me with it?
This is the codesandbox for the issue [https://xf6ykt.csb.app/](https://xf6ykt.csb.app/) 
Any help will be appreciated.

**Comment by leapful** [comment]
Closed as duplicated with discussion https://github.com/react-hook-form/react-hook-form/discussions/10134.
